lib/butler/adapters/console.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Butler.Adapters.Console do
+  require Logger
+
+  def start_link(_opts \\ []) do
```

Comment:
Yup. Just generic params that some of the adapters might need to have. We don't need them for the console adapter hence the `_`